tensorboard/plugins/histogram/summary_v2.py

```diff
@@ -41,6 +41,36 @@
 def histogram(name, data, step=None, buckets=None, description=None):
     """Write a histogram summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
+
+    Logs a histogram to the current log directory for later analysis in
+    TensorBoard's 'Histograms' and 'Distributions' dashboards (one histogram
+    will appear in both). Like `tf.summary.scalar` points, each histogram is
+    associated with a `step` and a `name`. All the histograms with the same
+    `name` constitute a time series of histograms.
+
+    This example writes 2 histograms:
+
+    ```python
+    test_summary_writer = tf.summary.create_file_writer('test/logs')
+    with test_summary_writer.as_default():
+      tf.summary.histogram("activations", [-10, 0, 10], step=0)
+      tf.summary.histogram("initial weights", tf.random.normal([5]), step=0)
+    ```
+
+    A common use case is to examine the changing activation patterns (or lack
+    thereof) at specific layers in a neural network, over time.
+
+    ```python
+    test_summary_writer = tf.summary.create_file_writer('test/logs')
+    with test_summary_writer.as_default():
+      for step in steps:
```

Comment:
Would be a little nicer if this example was runnable code.  Could do that via e.g. s/steps/range(10)/, and then passing in something like `tf.random.normal([1000], mean=step, stddev=N)` where N is the layer number.  That gives you a more appealing result like this:

<img width="1008" alt="Screen Shot 2021-03-19 at 3 37 50 PM" src="https://user-images.githubusercontent.com/710113/111848741-13fbb880-88c9-11eb-9b37-6c260afabcd2.png">

Reply:
Nice idea, done.